website/api_versioned_docs/version-1.0/generating-proofs/proving-options.md

```diff
@@ -18,6 +18,8 @@ Users can choose between:
 2. [Local Proving]: Proofs are generated on the user's own CPU/GPU.
 3. [Remote Proving]: Proofs are generated by [Bonsai], a highly parallelized, highly performant proving service.
 
+Furthermore, [prove_with_opts](https://docs.rs/risc0-zkvm/latest/risc0_zkvm/trait.Prover.html#method.prove_with_opts) allows users to choose between [composite, succinct or groth16 receipts](https://docs.rs/risc0-zkvm/latest/risc0_zkvm/enum.ReceiptKind.html).
```

Comment:
also, please links to the bottom of the doc to make it easier to manage link maintenance.

Reply:
ty! ✅